### PR TITLE
Feature/netlify

### PR DESCRIPTION
**Pull request by janssejl**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "start": "./node_modules/.bin/ng serve",
     "start-local": "./node_modules/.bin/ng serve --environment local",
     "environment-vars": "node ./process-template.js",
-    "build": "./node_modules/.bin/ng build --environment $CRDS_BUILD_ENV --base-href=/group-leader --prod --output-hashing=none --aot false ",
+    "build": "./node_modules/.bin/ng build --environment $CRDS_BUILD_ENV --base-href=/group-leader/ --prod --output-hashing=none --aot false ",
```
Comment (Doug-Shannon) [inline package.json]: how are you handling the $CRDS_BUILD_ENV?  Are you setting this somewhere in the netlify ui?

File: .angular-cli.json [hunk 2]
```diff
@@ -9,19 +9,30 @@
...
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
-      "styles": [
-        "styles.scss"
+      "styles": ["styles.scss"],
+      "scripts": [
+        "../node_modules/jquery/dist/jquery.min.js",
+        "../node_modules/bootstrap/dist/js/bootstrap.min.js",
+        "../node_modules/imgix.js/dist/imgix.min.js",
+        "../node_modules/crds-shared-header/dist/crds-shared-header-v0.7.2.min.js"
```
[the remark this answers is not in the document]
Comment (janssejl) [reply .angular-cli.json]: removed scripts. I think I had this when testing the crds-shared-header and forgot to remove

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -10,24 +10,20 @@ import { Router } from '@angular/router';
...
-  apiEndpoint = environment.apiEndpoint;
-  viewReady: boolean = false;
-  submitted: boolean = false;
-  buttonText: string = 'Start';
+  private openForBusiness: boolean = true;
+  private apiEndpoint = environment.apiEndpoint;
+  private viewReady: boolean = false;
+  private submitted: boolean = false;
+  private buttonText: string = 'Start';
+  private contentReady: boolean = false;
```
[the remark this answers is not in the document]
Comment (janssejl) [reply src/app/home/home.component.ts]: made public

File: package.json (hunk 1 above)
Comment (janssejl) [reply package.json]: yes they are in the netlify UI

**Pull request merged**